packages/driver/src/cypress/error_messages.ts

```diff
@@ -853,11 +853,11 @@ export default {
     },
     invalid_new_query: {
       message: '`Cypress.Commands.addQuery()` is used to create new queries, but `{{name}}` is an existing Cypress command or query, or is reserved internally by Cypress.\n\n If you want to override an existing command or query, use `Cypress.Commands.overrideQuery()` instead.',
-      docsUrl: 'https://on.cypress.io/custom-commands',
+      docsUrl: 'https://on.cypress.io/custom-queries',
```

Comment:
This is a new `on` link, for the new page I'm adding in the docs PR. I'll open a services PR to add it shortly.